src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/customers/CustomerSyncIT.java

```diff
@@ -0,0 +1,147 @@
+import io.sphere.sdk.customers.CustomerDraftBuilder;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.models.ResourceIdentifier;
+import io.sphere.sdk.stores.Store;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.commercetools.sync.commons.asserts.statistics.AssertionsForStatistics.assertThat;
+import static com.commercetools.sync.customers.utils.CustomerReferenceResolutionUtils.buildCustomerQuery;
+import static com.commercetools.sync.customers.utils.CustomerReferenceResolutionUtils.mapToCustomerDrafts;
+import static com.commercetools.sync.integration.commons.utils.CustomerITUtils.createSampleCustomerJaneDoe;
+import static com.commercetools.sync.integration.commons.utils.CustomerITUtils.createSampleCustomerJohnDoe;
+import static com.commercetools.sync.integration.commons.utils.CustomerITUtils.deleteCustomerSyncTestData;
+import static com.commercetools.sync.integration.commons.utils.ITUtils.createCustomFieldsJsonMap;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_TARGET_CLIENT;
+import static com.commercetools.sync.integration.commons.utils.StoreITUtils.createStore;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CustomerSyncIT {
+    private List<String> errorMessages;
+    private List<Throwable> exceptions;
+    private CustomerSync customerSync;
+
+    @BeforeEach
+    void setup() {
+        deleteCustomerSyncTestDataFromProjects();
+
+        createSampleCustomerJohnDoe(CTP_SOURCE_CLIENT);
+        createSampleCustomerJaneDoe(CTP_SOURCE_CLIENT);
+
+        createSampleCustomerJohnDoe(CTP_TARGET_CLIENT);
+
+        setUpCustomerSync();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        deleteCustomerSyncTestDataFromProjects();
+    }
+
+    private static void deleteCustomerSyncTestDataFromProjects() {
+        deleteCustomerSyncTestData(CTP_SOURCE_CLIENT);
+        deleteCustomerSyncTestData(CTP_TARGET_CLIENT);
+    }
+
+    private void setUpCustomerSync() {
+        errorMessages = new ArrayList<>();
+        exceptions = new ArrayList<>();
+        final CustomerSyncOptions customerSyncOptions = CustomerSyncOptionsBuilder
+            .of(CTP_TARGET_CLIENT)
+            .errorCallback((exception, oldResource, newResource, actions) -> {
+                errorMessages.add(exception.getMessage());
+                exceptions.add(exception);
+            })
+            .build();
+        customerSync = new CustomerSync(customerSyncOptions);
+    }
+
+    @Test
+    void sync_WithoutUpdates_ShouldReturnProperStatistics() {
```

Comment:
In the other CustomerSyncIT, you have 2 different cases: `sync_WithNewCustomer_ShouldCreateCustomer` and `sync_WithSameCustomer_ShouldNotUpdateCustomer` -  should you have also these cases here?

Reply:
I could add but in the end, the examples in integration tests cover all, less it better to not increase built time.